dbx/commands/launch.py

```diff
@@ -327,6 +334,16 @@ def launch(self) -> Tuple[Dict[Any, Any], Optional[str]]:
         job_data = matching_jobs[0]
         job_id = job_data["job_id"]
 
+        is_multi_task = False
+        try:
+            is_multi_task = job_data["settings"]["format"] == "MULTI_TASK"
+        except KeyError:
+            pass
```

Comment:
could you please make it a bit simpler, for instance, this one-liner?

```
is_multi_task = job_data.get("settings", {}).get("format") == "MULTI_TASK"
```
?

Reply:
Nice! Will do 👍